internal/controller/mutatingwebhookconfiguration/controller.go

```diff
@@ -55,8 +58,14 @@ func NewReconciler(client client.Client, certProvider *certificate.Provider, nam
 }
 
 func (r *Reconciler) SetupWithManager(mgr ctrl.Manager, options controller.Options) error {
+	kind := reflect.TypeOf(admissionregistrationv1.MutatingWebhookConfiguration{}).Name()
```

Comment:
Using reflect to do this unnecessary because we know what the type is. In general, I like to not use reflect where possible due to performance implications.